truffle/com.oracle.truffle.api.debug/src/com/oracle/truffle/api/debug/Debugger.java

```diff
@@ -957,15 +957,9 @@ public FrameInstance visitFrame(FrameInstance frameInstance) {
                     if (stackIndex < contextStackDepth) {
                         final Node callNode = frameInstance.getCallNode();
                         if (callNode != null) {
```

Comment:
It seems that changes in frame iteration have led to this issue.  Is it now stable?